NFO:werkzeug:←[31m←[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.←[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.16.18.21:5000
INFO:werkzeug:←[33mPress CTRL+C to quit←[0m
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:17] "←[32mGET / HTTP/1.1←[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:17] "←[32mGET /dashboard/ HTTP/1.1←[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:17] "GET /dashboard/engineer HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:17] "←[36mGET /static/cully.png HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:17] "←[36mGET /static/css/form.css HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:20] "GET /reports/new HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:20] "←[36mGET /static/css/form.css HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:20] "←[36mGET /static/cully.png HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:20] "GET /dashboard/db-status HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:21] "GET /reports/new/sat/full HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:21] "←[36mGET /static/css/form.css HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:21] "←[36mGET /static/cully.png HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:21] "←[36mGET /static/Cully_Watermark.jpg HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:21] "←[36mGET /static/js/form.js HTTP/1.1←[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:22] "GET /api/get-users-by-role HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:44] "GET /refresh_csrf HTTP/1.1" 200 -
send: 'ehlo Cullyaz-dev2022.ad.cullyautomation.net\r\n'
reply: b'250-smtp.gmail.com at your service, [52.169.156.168]\r\n'
reply: b'250-SIZE 35882577\r\n'
reply: b'250-8BITMIME\r\n'
reply: b'250-STARTTLS\r\n'
reply: b'250-ENHANCEDSTATUSCODES\r\n'
reply: b'250-PIPELINING\r\n'
reply: b'250-CHUNKING\r\n'
reply: b'250 SMTPUTF8\r\n'
reply: retcode (250); Msg: b'smtp.gmail.com at your service, [52.169.156.168]\nSIZE 35882577\n8BITMIME\nSTARTTLS\nENHANCEDSTATUSCODES\nPIPELINING\nCHUNKING\nSMTPUTF8'
send: 'STARTTLS\r\n'
reply: b'220 2.0.0 Ready to start TLS\r\n'
reply: retcode (220); Msg: b'2.0.0 Ready to start TLS'
send: 'ehlo Cullyaz-dev2022.ad.cullyautomation.net\r\n'
reply: b'250-smtp.gmail.com at your service, [52.169.156.168]\r\n'
reply: b'250-SIZE 35882577\r\n'
reply: b'250-8BITMIME\r\n'
reply: b'250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH\r\n'
reply: b'250-ENHANCEDSTATUSCODES\r\n'
reply: b'250-PIPELINING\r\n'
reply: b'250-CHUNKING\r\n'
reply: b'250 SMTPUTF8\r\n'
reply: retcode (250); Msg: b'smtp.gmail.com at your service, [52.169.156.168]\nSIZE 35882577\n8BITMIME\nAUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH\nENHANCEDSTATUSCODES\nPIPELINING\nCHUNKING\nSMTPUTF8'
send: 'AUTH PLAIN AG1lZGEucmV2YW50aEBnbWFpbC5jb20AcnFyY2JhbmpydXVjaGp2bQ==\r\n'
reply: b'535-5.7.8 Username and Password not accepted. For more information, go to\r\n'
reply: b'535 5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp\r\n'
reply: retcode (535); Msg: b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp'
send: 'AUTH LOGIN bWVkYS5yZXZhbnRoQGdtYWlsLmNvbQ==\r\n'
reply: b'334 UGFzc3dvcmQ6\r\n'
reply: retcode (334); Msg: b'UGFzc3dvcmQ6'
send: 'cnFyY2JhbmpydXVjaGp2bQ==\r\n'
reply: b'535-5.7.8 Username and Password not accepted. For more information, go to\r\n'
reply: b'535 5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp\r\n'
reply: retcode (535); Msg: b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp'
send: 'QUIT\r\n'
reply: b'221 2.0.0 closing connection ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp\r\n'
reply: retcode (221); Msg: b'2.0.0 closing connection ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp'
ERROR:utils:Email attempt 1/3 failed: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp')
Traceback (most recent call last):
  File "E:\report generator\WORKINGCULLY (1)\WORKINGCULLY\utils.py", line 372, in send_email
    server.login(smtp_username, smtp_password)
  File "C:\Program Files\Python312\Lib\smtplib.py", line 750, in login
    raise last_exception
  File "C:\Program Files\Python312\Lib\smtplib.py", line 739, in login
    (code, resp) = self.auth(
                   ^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\smtplib.py", line 662, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-3cf3458a67esm23046047f8f.62 - gsmtp')
send: 'ehlo Cullyaz-dev2022.ad.cullyautomation.net\r\n'
reply: b'250-smtp.gmail.com at your service, [52.169.156.168]\r\n'
reply: b'250-SIZE 35882577\r\n'
reply: b'250-8BITMIME\r\n'
reply: b'250-STARTTLS\r\n'
reply: b'250-ENHANCEDSTATUSCODES\r\n'
reply: b'250-PIPELINING\r\n'
reply: b'250-CHUNKING\r\n'
reply: b'250 SMTPUTF8\r\n'
reply: retcode (250); Msg: b'smtp.gmail.com at your service, [52.169.156.168]\nSIZE 35882577\n8BITMIME\nSTARTTLS\nENHANCEDSTATUSCODES\nPIPELINING\nCHUNKING\nSMTPUTF8'
send: 'STARTTLS\r\n'
reply: b'220 2.0.0 Ready to start TLS\r\n'
reply: retcode (220); Msg: b'2.0.0 Ready to start TLS'
send: 'ehlo Cullyaz-dev2022.ad.cullyautomation.net\r\n'
reply: b'250-smtp.gmail.com at your service, [52.169.156.168]\r\n'
reply: b'250-SIZE 35882577\r\n'
reply: b'250-8BITMIME\r\n'
reply: b'250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH\r\n'
reply: b'250-ENHANCEDSTATUSCODES\r\n'
reply: b'250-PIPELINING\r\n'
reply: b'250-CHUNKING\r\n'
reply: b'250 SMTPUTF8\r\n'
reply: retcode (250); Msg: b'smtp.gmail.com at your service, [52.169.156.168]\nSIZE 35882577\n8BITMIME\nAUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH\nENHANCEDSTATUSCODES\nPIPELINING\nCHUNKING\nSMTPUTF8'
send: 'AUTH PLAIN AG1lZGEucmV2YW50aEBnbWFpbC5jb20AcnFyY2JhbmpydXVjaGp2bQ==\r\n'
reply: b'535-5.7.8 Username and Password not accepted. For more information, go to\r\n'
reply: b'535 5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp\r\n'
reply: retcode (535); Msg: b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp'
send: 'AUTH LOGIN bWVkYS5yZXZhbnRoQGdtYWlsLmNvbQ==\r\n'
reply: b'334 UGFzc3dvcmQ6\r\n'
reply: retcode (334); Msg: b'UGFzc3dvcmQ6'
send: 'cnFyY2JhbmpydXVjaGp2bQ==\r\n'
reply: b'535-5.7.8 Username and Password not accepted. For more information, go to\r\n'
reply: b'535 5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp\r\n'
reply: retcode (535); Msg: b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp'
send: 'QUIT\r\n'
reply: b'221 2.0.0 closing connection 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp\r\n'
reply: retcode (221); Msg: b'2.0.0 closing connection 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp'
ERROR:utils:Email attempt 2/3 failed: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp')
Traceback (most recent call last):
  File "E:\report generator\WORKINGCULLY (1)\WORKINGCULLY\utils.py", line 372, in send_email
    server.login(smtp_username, smtp_password)
  File "C:\Program Files\Python312\Lib\smtplib.py", line 750, in login
    raise last_exception
  File "C:\Program Files\Python312\Lib\smtplib.py", line 739, in login
    (code, resp) = self.auth(
                   ^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\smtplib.py", line 662, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45c6faad9cfsm37872185e9.0 - gsmtp')
send: 'ehlo Cullyaz-dev2022.ad.cullyautomation.net\r\n'
reply: b'250-smtp.gmail.com at your service, [52.169.156.168]\r\n'
reply: b'250-SIZE 35882577\r\n'
reply: b'250-8BITMIME\r\n'
reply: b'250-STARTTLS\r\n'
reply: b'250-ENHANCEDSTATUSCODES\r\n'
reply: b'250-PIPELINING\r\n'
reply: b'250-CHUNKING\r\n'
reply: b'250 SMTPUTF8\r\n'
reply: retcode (250); Msg: b'smtp.gmail.com at your service, [52.169.156.168]\nSIZE 35882577\n8BITMIME\nSTARTTLS\nENHANCEDSTATUSCODES\nPIPELINING\nCHUNKING\nSMTPUTF8'
send: 'STARTTLS\r\n'
reply: b'220 2.0.0 Ready to start TLS\r\n'
reply: retcode (220); Msg: b'2.0.0 Ready to start TLS'
send: 'ehlo Cullyaz-dev2022.ad.cullyautomation.net\r\n'
reply: b'250-smtp.gmail.com at your service, [52.169.156.168]\r\n'
reply: b'250-SIZE 35882577\r\n'
reply: b'250-8BITMIME\r\n'
reply: b'250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH\r\n'
reply: b'250-ENHANCEDSTATUSCODES\r\n'
reply: b'250-PIPELINING\r\n'
reply: b'250-CHUNKING\r\n'
reply: b'250 SMTPUTF8\r\n'
reply: retcode (250); Msg: b'smtp.gmail.com at your service, [52.169.156.168]\nSIZE 35882577\n8BITMIME\nAUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH\nENHANCEDSTATUSCODES\nPIPELINING\nCHUNKING\nSMTPUTF8'
send: 'AUTH PLAIN AG1lZGEucmV2YW50aEBnbWFpbC5jb20AcnFyY2JhbmpydXVjaGp2bQ==\r\n'
reply: b'535-5.7.8 Username and Password not accepted. For more information, go to\r\n'
reply: b'535 5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp\r\n'
reply: retcode (535); Msg: b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp'
send: 'AUTH LOGIN bWVkYS5yZXZhbnRoQGdtYWlsLmNvbQ==\r\n'
reply: b'334 UGFzc3dvcmQ6\r\n'
reply: retcode (334); Msg: b'UGFzc3dvcmQ6'
send: 'cnFyY2JhbmpydXVjaGp2bQ==\r\n'
reply: b'535-5.7.8 Username and Password not accepted. For more information, go to\r\n'
reply: b'535 5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp\r\n'
reply: retcode (535); Msg: b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp'
send: 'QUIT\r\n'
reply: b'221 2.0.0 closing connection 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp\r\n'
reply: retcode (221); Msg: b'2.0.0 closing connection 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp'
ERROR:utils:Email attempt 3/3 failed: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp')
Traceback (most recent call last):
  File "E:\report generator\WORKINGCULLY (1)\WORKINGCULLY\utils.py", line 372, in send_email
    server.login(smtp_username, smtp_password)
  File "C:\Program Files\Python312\Lib\smtplib.py", line 750, in login
    raise last_exception
  File "C:\Program Files\Python312\Lib\smtplib.py", line 739, in login
    (code, resp) = self.auth(
                   ^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\smtplib.py", line 662, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-45b74950639sm283685925e9.17 - gsmtp')
Would send edit link to rmeda@cully.ie: 19890bcf-9093-4e33-9079-465e3888d2d8
INFO:werkzeug:127.0.0.1 - - [03/Sep/2025 10:02:51] "POST /generate HTTP/1.1" 200 -